src/miner_lora.erl

```diff
@@ -247,6 +247,9 @@ handle_call({send, Payload, When, ChannelSelectorFun, DataRate, Power, IPol}, Fr
                              <<"data">> => base64:encode(Payload)
                             }
                         }),
+
+            lager:info("PULL_RESP ~p to ~p:~p", [jsx:decode(BinJSX), IP, Port]),
```

Comment:
as BinJSX is encoded in the previous line, it would be good to declare the unencoded version as a new var and use that in the lager statement...save having to decode a payload we  already have in unencoded state

Reply:
Ah good point. Just pushed a commit to remedy